### PR TITLE
Add support for the TaxId resource and APIs

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

**Comment by remi-stripe** [comment]
This is now ready to be reviewed. 

r? @mickjermsurawong-stripe

File: lib/resources/TaxIds.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * TaxIds is a unique resource in that, upon instantiation,
+ * requires a customerId, and therefore each of its methods only
+ * require the taxId argument.
+ *
+ * This streamlines the API specifically for the case of accessing a tax id
+ * on a returned customer object.
+ *
+ * E.g. customerObject.tax_id.retrieve(taxIdId)
```
Comment (mickjermsurawong-stripe) [inline lib/resources/TaxIds.js]: should this be `customerObject.tax_ids` (the id[s])?

**Comment by mickjermsurawong-stripe** [comment]
ptal @remi-stripe

File: lib/resources/TaxIds.js (hunk 1 above)
Comment (remi-stripe) [reply lib/resources/TaxIds.js]: I'll be honest, this comment makes little sense to me (the library really needs to be cleaned-up/rewritten :p) but likely

**Comment by remi-stripe** [comment]
@mickjermsurawong-stripe Nice catch. Fixed. PTAL

**Comment by mickjermsurawong-stripe** [comment]
LGTM 
thanks Remi!

**Pull request merged**